### ISSUE TITLE
Donations

### ISSUE THREAD

**Issue by treequell** [davidtakac/prognoza#59]
Do you accept donations?

**Comment by davidtakac** [comment]
No, but I would like to 😄 I'll see about opening a PayPal donation thing soon. Thank you!

**Comment by davidtakac** [comment]
I just created a [LiberaPay account](https://liberapay.com/davidtakac) so you can donate there if you wish. I also added the link to the Sponsorship section of the repository.
